curriculum/challenges/_meta/registration-form/meta.json

```diff
@@ -0,0 +1,242 @@
+{
+  "name": "Registration Form",
+  "isUpcomingChange": true,
+  "dashedName": "registration-form",
+  "order": 42,
```

Comment:
🤷‍♂️

Reply:
I would maybe change it since 42 is the default. Well, either this one or the accessibility quiz will have to change - otherwise...

"
> The result was that the lessons jumped between the two blocks after finishing each lesson.
"Tom"

"
-Shaun